docs/release-notes/release-notes-0.20.1.md

```diff
@@ -92,6 +92,12 @@
   safe single-writer behavior until the wallet subsystem is fully 
   concurrent-safe.
 
+* [Add caching for](https://github.com/lightningnetwork/lnd/pull/10363)
+`IsPublicNode` query which speedup calls to check for nodes visibility status.
```

Comment:
I wonder if we can also add this cache for the KV store, it shouldn't be a big lift but at least it makes the config value more general, because right now kv db node runners would also think they have this feature available. So either we add clear comments that this is a sql feature or we also add it to the kv store which probably isn't a big change and would improve the performance there as well